package.json

```diff
@@ -26,7 +26,8 @@
         "onCommand:aws.invokeLambda",
         "onCommand:aws.deployLambda",
         "onCommand:aws.getLambdaConfig",
-        "onCommand:aws.getLambdaPolicy"
+        "onCommand:aws.getLambdaPolicy",
+        "onDebug"
```

Comment:
Good catch--I thought that we couldn't use this because we also care about adding configs to an existing `launch.json`. But it turns out that this is only supported for static configs that appear in `package.json`'s `contributes.debuggers.*.configurationSnippets` property.

✔️